docs/src/main/asciidoc/user-guide/topics/import_files_to_panel.adoc

```diff
@@ -0,0 +1,284 @@
+
+An *Open File* dialog appears.  Select the location for JSONSchema.json.
+
+image:ImportJSON2.png[Import a JSON File to Source Panel - Open File Dialog]
+
+The fields now appear in the *Source* panel as you've defined them in the imported JSON schema.
+
+image:ImportJSON3.png[Import a JSON File to Source Panel - Import Complete]
+
+Now lets import an XML schema into the *Target* panel.  
+
+* XML Schema File: *XMLSchema.xml*
+
+```
+<d:SchemaSet xmlns:d="http://atlasmap.io/xml/schemaset/v2" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <xsd:schema targetNamespace="http://syndesis.io/v1/swagger-connector-template/request" elementFormDefault="qualified">
+      <xsd:element name="request">
+        <xsd:complexType>
+          <xsd:sequence>
+            <xsd:element name="body">
+              <xsd:complexType>
+                <xsd:sequence>
+                  <xsd:element ref="Pet" />
+                </xsd:sequence>
+              </xsd:complexType>
+            </xsd:element>
+          </xsd:sequence>
+        </xsd:complexType>
+      </xsd:element>
+    </xsd:schema>
+    <d:AdditionalSchemas>
+      <xsd:schema>
+        <xsd:element name="Pet">
+          <xsd:complexType>
+            <xsd:sequence>
+              <xsd:element name="id" type="xsd:decimal" />
+              <xsd:element name="Category">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="id" type="xsd:decimal" />
+                    <xsd:element name="name" type="xsd:string" />
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="name" type="xsd:string" />
+              <xsd:element name="photoUrl">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="photoUrl" type="xsd:string" maxOccurs="unbounded" minOccurs="0" />
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="tag">
+                <xsd:complexType>
+                  <xsd:sequence>
+                    <xsd:element name="Tag" maxOccurs="unbounded" minOccurs="0">
+                      <xsd:complexType>
+                        <xsd:sequence>
+                          <xsd:element name="id" type="xsd:decimal" />
+                          <xsd:element name="name" type="xsd:string" />
+                        </xsd:sequence>
+                      </xsd:complexType>
+                    </xsd:element>
+                  </xsd:sequence>
+                </xsd:complexType>
+              </xsd:element>
+              <xsd:element name="status" type="xsd:string" />
+            </xsd:sequence>
+          </xsd:complexType>
+        </xsd:element>
+      </xsd:schema>
+    </d:AdditionalSchemas>
+  </d:SchemaSet>
+```
+As was done in the previous JSON file import, select the import icon this time in the *Target* panel.
```

Comment:
Instead: In the data mapper, click the import icon at the top of the *Target* panel.